dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/apikey/HibernateApiTokenStore.java

```diff
@@ -55,16 +57,18 @@ public HibernateApiTokenStore( SessionFactory sessionFactory, JdbcTemplate jdbcT
     }
 
     @Override
-    public List<ApiToken> getAllOwning( User currentUser )
+    @Nonnull
+    public List<ApiToken> getAllOwning( @Nonnull User currentUser )
```

Comment:
Just a minor comment. This method could be renamed to `findByCreatedBy( User user )`